web/src/pages/setting/SettingContent.vue

```diff
@@ -0,0 +1,98 @@
+<script lang="ts" setup>
```

Comment:
Should probably rename this to SettingsModal or something like that. 

It's also not a page at the moment, so you should move it to components.

Pages is reserved for actual routes and subroutes.